### PR TITLE
CAN IO code for hands

### PR DESCRIPTION
**Pull request by antoinealb**

### CODE REVIEW

File: master-firmware/src/main.c [hunk 1]
```diff
@@ -268,6 +268,15 @@ void init_base_motors(void)
     motor_manager_create_driver(&motor_manager, "right-wheel");
 }
 
+
+void init_hands(void)
+{
+    bus_enumerator_add_node(&bus_enumerator, "left-fingers", NULL);
+    bus_enumerator_add_node(&bus_enumerator, "left-sensors", NULL);
+    bus_enumerator_add_node(&bus_enumerator, "right-fingers", NULL);
```
Comment (antoinealb) [inline master-firmware/src/main.c]: @nuft why is there both left-sensors and left-fingers ?

Comment (nuft) [inline master-firmware/src/main.c]: initially, we planned two io-boards per hand: one for controlling the fingers and one for the sensors

Comment (nuft) [inline master-firmware/src/main.c]: good catch, we can update this to 'left-hand', 'right-hand'.

Comment (antoinealb) [inline master-firmware/src/main.c]: Oh right, I thought it was in a recent commit.

**Pull request merged**